recipes/protobuf/all/conandata.yml

```diff
@@ -1,9 +1,18 @@
 sources:
```

Comment:
Please, if there is no strong need to add more patch releases, then follow the policy we have defined: keep only the latest minor version. In this case you should add `3.19.4` and remove `3.19.3`, `3.19.2` and `3.19.1`.

If we only keep adding versions, compilation times will explode for each pull-request.

Thanks!

Reply:
Indeed, good to know. Will update the PR accordingly.

Reply:
Thanks! I really want to update some consumers of `protobuf`, so please, if you can prepare this PR as suggested it would be great.